lib/open_api_spex/cast/string.ex

```diff
@@ -30,6 +30,77 @@ defmodule OpenApiSpex.Cast.String do
     end
   end
 
+  @uuid_valid_characters [
```

Comment:
Should upper case be allowed, `Ecto.UUID.cast` seems accepts them:

```
iex(1)> Ecto.UUID.cast("9F73C262-143C-4BB6-9EF3-67997A927ED5")
{:ok, "9f73c262-143c-4bb6-9ef3-67997a927ed5"}
```

Reply:
@mbuhot Good catch, yes. Updated the code and added a test for that.